instance/models/database_server.py

```diff
@@ -120,6 +121,10 @@ class DatabaseServer(ValidateModelMixin, TimeStampedModel):
     class Meta:
         abstract = True
 
+    # Human readable identifier for database instances
+    name = models.CharField(max_length=128, unique=True)
+    description = models.CharField(max_length=128, blank=True)
+
     # Host name or IP address of this database server
     hostname = models.CharField(max_length=128, unique=True)
```

Comment:
@bdero Also not your fault, but looking at the field in other places made me realize that we use `hostname` in some places, and `domain` in other places (instances, load balancer), with varying lengths, so it would likely make sense to standardize this too. Maybe that could go into the `NamedHostMixin` or `NamedDomainMixin` (depending on the one you chose - although given that instances use `domain`, it would like be a lot less trouble to go with that one).